### PR TITLE
Bug 1190683 - 'start communicating' phrase is changed

### PR DESCRIPTION
**Pull request by ruoslepov**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fdd83074ad8e5f1ff5628a3c2ee57ab113e7c43f

File: apps/communications/dialer/locales/dialer.en-US.properties [hunk 1]
```diff
@@ -57,7 +57,7 @@ mmi-notification-title-with-sim = ({{sim}}) {{title}}
 mute.ariaLabel=Mute
 newCall.ariaLabel=New call
 no-logs-msg-1=No calls recorded
-no-logs-msg-2=Start communicating now
+no-logs-msg-2=Start a conversation now
```
Comment (zbraniecki) [inline apps/communications/dialer/locales/dialer.en-US.properties]: yup, please, bump the ID to let localizers know the string has changed.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ca6620f212bb196d04f0f1f4386d06f015c74a71

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=406f7984cb7bf535c8da9e0fcbd12b7a20d7c95f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=50e6108a5abb0c97dbc9e50e3248707c967f5879

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=16906bf0406231a3fab89dba5a08a9b2ec47650b

**Pull request merged**